package.json

```diff
@@ -54,6 +54,7 @@
         "all-contributors-cli": "^6.26.1",
         "autoprefixer": "^10.4.18",
         "cypress": "^13.10.0",
+        "cypress-plugin-tab": "^1.0.5",
```

Comment:
what a dumb plugin need 🤣  I read how people have been waiting since 2017 for the ability to tab so this still plugin still exists